int32[10] confidence
float32[10] dist
int32[10] angle
int32 num_lines
